builtin/providers/azurerm/resource_arm_dns_zone.go

```diff
@@ -0,0 +1,122 @@
+package azurerm
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/hashicorp/terraform/helper/schema"
+	"github.com/jen20/riviera/dns"
+)
+
+func resourceArmDnsZone() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceArmDnsZoneCreate,
+		Read:   resourceArmDnsZoneRead,
+		Update: resourceArmDnsZoneCreate,
+		Delete: resourceArmDnsZoneDelete,
+
+		Schema: map[string]*schema.Schema{
+			"name": &schema.Schema{
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+
+			"resource_group_name": &schema.Schema{
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+
+			"number_of_record_sets": &schema.Schema{
+				Type:     schema.TypeString,
+				Optional: true,
+				Computed: true,
+			},
+
+			"max_number_of_record_sets": &schema.Schema{
+				Type:     schema.TypeString,
+				Optional: true,
+				Computed: true,
+			},
+		},
+	}
+}
+
+func resourceArmDnsZoneCreate(d *schema.ResourceData, meta interface{}) error {
+	client := meta.(*ArmClient)
+	rivieraClient := client.rivieraClient
+
+	createRequest := rivieraClient.NewRequest()
+	createRequest.Command = &dns.CreateDNSZone{
+		Name:              d.Get("name").(string),
+		Location:          "global",
+		ResourceGroupName: d.Get("resource_group_name").(string),
+	}
+
+	createResponse, err := createRequest.Execute()
+	if err != nil {
+		return fmt.Errorf("Error creating DNS Zone: %s", err)
+	} else if !createResponse.IsSuccessful() {
```

Comment:
Correct - I thought I had changed that already but apparently not. New commit pushed rectifying all the occurrences of this.